docs/build/windows.md

```diff
@@ -150,28 +123,36 @@ PowerShell.
 # for *much* faster download, hide progress bar (PowerShell/PowerShell#2138)
 $ProgressPreference = 'SilentlyContinue'
 choco install git jq python ninja pandoc meson
-refreshenv
 ```
 
 **Environment variables**:
-* [`KEYMAN_ROOT`](#keyman_root)
-* `PATH`: add your Python scripts folder to your path: it will normally be `%appdata%\Python\Python310\Scripts`.
+
+If you pull the entire `keyman.git` repo to `c:\keyman`, then the paths by
+default will work without changes. Otherwise, you will need to set an
+environment variable `KEYMAN_ROOT` to the root path of the Keyman repo. For
+example:
```

Comment:
This does not match our recommendation for the repo filesystem layout:

https://github.com/keymanapp/keyman/blob/75238dcbf7af3ebc4ae65c922abc7e094ca72079/docs/build/windows.md?plain=1#L37-L48

Shouldn't the default case, without `KEYMAN_ROOT`, work based off of that, rather than a different arbitrary path?

----

FWIW - I use neither location and utilize the `KEYMAN_ROOT` variable on my machine.  I like `c:/keymanapp/keyman`, mirroring GitHub's URL structure for it and all our other repos.  Not saying to change it; just saying that I "dogfood" the use of `KEYMAN_ROOT`.  Do we have someone "dogfooding" use of the default, without `KEYMAN_ROOT`?

Reply:
It's historical and at this point I usually just say 'use `KEYMAN_ROOT`' but I kept the docs there for docs.